packages/nocodb/src/services/bulk-data-alias.service.ts

```diff
@@ -44,12 +44,22 @@ export class BulkDataAliasService {
     param: PathParams & {
       body: any;
       cookie: any;
+      chunkSize?: number;
+      foreign_key_checks?: boolean;
+      raw?: boolean;
```

Comment:
raw: this option is to skip alias - column name mapping, before/after hooks, data validations. this is meant for internal usage (duplicate, export/import)
foreign_key_checks: if this option is true we skip foreign key checks on that transaction, allowing us to insert belongsTo records in place as foreign keys without waiting linked record to be created which allow us to avoid one extra iteration of update calls over those records.

Reply:
please include these infos in code :D